lib/child_process.js

```diff
@@ -417,12 +417,12 @@ function setupChannel(target, channel) {
         message.type = 'net.Socket';
       } else if (handle instanceof net.Server) {
         message.type = 'net.Server';
-      } else if (handle instanceof process.binding('tcp_wrap').TCP ||
-                 handle instanceof process.binding('pipe_wrap').Pipe) {
+      } else if (handle instanceof handleWraps.TCP ||
```

Comment:
[here](https://github.com/iojs/io.js/blob/v1.x/lib/child_process.js#L19) and the function below, it's a `Getter`